nfsn-cron.sh

```diff
@@ -17,8 +17,8 @@ if find certs -name cert.pem -type l \
 then
 	echo " + Certificte will expire in 30 days or less! SSH into this site and"
 	echo "   run the following commands to renew your certificates:"
-	echo "	cd ~/lets-nfsn.sh/letsencrypt.sh/"
-	echo "	./letsencrypt.sh --cron"
+	printf '	cd %q\n' "$PWD"
+	echo   '	./letsencrypt.sh --cron'
```

Comment:
It very happily does; on my servers it gets installed to `/usr/bin/`, reads global config in `/etc/`, and runs from wherever you please.